modules/core/src/main/java/org/apache/ignite/internal/management/baseline/AbstractBaselineCommand.java

```diff
@@ -57,7 +57,7 @@ public abstract class AbstractBaselineCommand implements ComputeCommand<Baseline
         BaselineTaskResult res,
         Consumer<String> printer
     ) {
-        printer.accept("Cluster state: " + (res.isActive() ? "active" : "inactive"));
+        printer.accept("Cluster state: " + res.clusterState().toString().toLowerCase().replace("_", " "));
```

Comment:
Do we have a strict requirement to leave words "active" and "inactive" in output as they are?
Can they be changed to show more precise Cluster state like this?

Cluster state: ACTIVE
Cluster state: INACTIVE
Cluster state: ACTIVE_READ_ONLY

Reply:
Cluster state output simplified (same as Enum values)